OJP/OJP_Trips.xsd

```diff
@@ -1062,6 +1062,18 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:group name="TripRefinementDataFilterGroup">
+		<xs:annotation>
+			<xs:documentation>Request data filters for Refinement</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
```

Comment:
In the workflow we discussed, we only had a single operator passed in the refinement. Do you have an usecase when a sequence is needed?

Reply:
It seems to be that with taxi, multiple companies may be choosen from.

Reply:
Matthias: There might exist aggregators. 
Malte: One trip result by operators? Currently a general taxi and  sharing, then update in the refinement. So I thought that refinement go to one operator. That's what expected. 
==> first do the chapter in the specification, discuss it, then finalize this PR (chapter 7.9)